app/app.go

```diff
@@ -460,11 +460,12 @@ func New(
 		appCodec,
 		keys[paymentmoduletypes.StoreKey],
 		keys[paymentmoduletypes.MemStoreKey],
-		app.GetSubspace(paymentmoduletypes.ModuleName),
 
 		app.BankKeeper,
 		app.AccountKeeper,
 		app.SpKeeper,
+
+		authtypes.NewModuleAddress(paymentmoduletypes.ModuleName).String(),
```

Comment:
we should use `govtypes.ModuleName` here

Reply:
fixed